docs/installation.rst

```diff
@@ -40,10 +40,224 @@ With that, you should be ready to use ``openfe``!
 Single file installer
 ~~~~~~~~~~~~~~~~~~~~~
 
-[COMING SOON!]
-
-.. TODO: maybe Mike can fill this in? just needs (1) how to download the
-   single file installer; (2) how to use the single file installer
+Single file installers are available for x86_64 Linux and MacOS. 
+They are attached to our `releases on GitHub <https://github.com/OpenFreeEnergy/openfe/releases>`_ and can be downloaded with a browser or ``curl`` (or similar tool) ::
+
+  $ curl -LOJ https://github.com/OpenFreeEnergy/openfe/releases/download/v0.7.1/OpenFEforge-0.7.1-Linux-x86_64.sh
+
+The single file installer contains all of the dependences required for ``openfe`` and does not require internet access to use.
+Both ``conda`` and ``mamba`` are also available in environment created by the single file installer and can be used to install additional packages.
```

Comment:
```suggestion
Both ``conda`` and ``mamba`` are also available in the environment created by the single file installer and can be used to install additional packages.
```
So the single file installers are really for folks that don't have external internet access, so it might not be that useful, but no problems in mentioning it.